python/xrprimer/transform/convention/world/mpl_world.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+from .base_world import BaseWorld
+
+
+class MatplotlibWorld(BaseWorld):
+    """A world class for SenseMoCap coordinate convention.
```

Comment:
Matplotlib coordinate convention?

Reply:
Fixed in [f973972](https://github.com/openxrlab/xrprimer/pull/39/commits/f9739722b97083b69836a4dc395c9cbabe4e62a6).